src/bundle/Resources/views/themes/admin/limitation/limitation_values.html.twig

```diff
@@ -36,6 +36,12 @@
 {% endapply %}
 {% endblock %}
 
+{% block ez_limitation_changeowner_value %}
```

Comment:
Is there any way to have this block name rebranded or does it need to follow the old namespace pattern?

Reply:
Unfortunately, unavoidable debt.